crates/core/tedge_watchdog/src/systemd_watchdog.rs

```diff
@@ -76,8 +76,9 @@ async fn start_watchdog_for_tedge_services(tedge_config_dir: PathBuf) {
         "tedge-mapper-aws",
         "tedge-mapper-collectd",
         "tedge-agent",
-        "c8y-log-plugin",
+        "tedge-log-plugin",
```

Comment:
Since the plan was to keep the `c8y-log-plugin` for one more release, we shouldn't be removing it from this list yet, right?

Reply:
Actually I'm in favour or releasing it as soon as possible because tedge-log-plugin is fairly simple, so I don't think there will be much harm in including it already.

Reply:
I wasn't contesting the inclusion of `tedge-log-plugin` but rather the **exclusion** of the `c8y-log-plugin`, while it's only deprecated but not fully removed.